HM9/HM9_task1.py

```diff
@@ -0,0 +1,102 @@
+class Any_company:
```

Comment:
in the class definition, we use CamleCase style.
AnyCompany

Reply:
fix